docs: reword local module info (#23557)
nuxt/nuxt: docs/2.guide/2.directory-structure/1.modules.md

When starting Nuxt, the `hello` module will be registered and the `/api/hello` route will be available.

> The local modules are registered by alphabetical order. You can change the order by prefixing with a number in front of each directory:

Reviewer: ```suggestion
Local modules are registered in alphabetical order. You can change the order by adding a number to the front of each directory name:
```
